internal/service/cluster/resource_cluster_test.go

```diff
@@ -1349,6 +1349,73 @@ func TestAccCluster_basicAWS_PausedToUnpaused(t *testing.T) {
 	})
 }
 
+func TestAccCluster_basic_RedactClientLogData(t *testing.T) {
+	var (
+		projectID   = acc.ProjectIDExecution(t)
+		clusterName = acc.RandomClusterName()
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             acc.CheckDestroyCluster,
+		Steps: []resource.TestStep{
+			{
+				Config: configRedactClientLogData(projectID, clusterName, nil),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "redact_client_log_data", "false"),
```

Comment:
I think if you duplicate this step and put it after step 3 the test will fail. Since it is `optional-computed` removing the value will see `no changes` and it will remain `true`

Reply:
yes, I think that's what we're doing in these cases.  Removing the field won't be flagged as a change so current value will stay